general-superstaq/general_superstaq/service.py

```diff
@@ -124,21 +124,24 @@ def submit_qubo(
         target: str,
         repetitions: int = 1000,
         method: Optional[str] = None,
-        maxout: Optional[int] = 1000,
+        max_solutions: Optional[int] = 1000,
     ) -> Dict[str, str]:
-        """Solves the QUBO given via the submit_qubo function in superstaq_client
+        """Solves the QUBO given via the submit_qubo function in superstaq_client, and returns any
+        number of specified dictionaries that seek the minimum of the energy landscape from the
+        given objective function known as output solutions.
 
         Args:
             qubo: A `qv.QUBO` object.
             target: The target to submit the qubo.
-            repetitions: Number of repetitions to use. Defaults to 1000.
-            method: An optional method parameter. Defaults to None.
-            maxout: An optional maxout paramter. Defaults to 1000.
+            repetitions: Number of times that the execution is repeated before stopping.
+            method: The parameter specifying method of QUBO solving execution. Defaults to none.
```

Comment:
probably better to say what the default method is - it's clear from the annotation that it defaults to None, but not clear what this means

Reply:
done.